tests/test_output_drivers.py

```diff
@@ -20,11 +22,24 @@
     ('float64', np.nan),
 ])
 def test_geotiff_outputs(tmpdir, dtype, nodata):
+    # Make a Test Metadata Type
+    class FakeMetadataType(MetadataType):
```

Comment:
I think it's time we start creating library for test code. Class like this one can be helpful in other tests.